extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -627,6 +621,16 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
                 return [_errorHookCnt, _autoExceptionInstrumented];
             };
             
+            function _resetAjaxAttempts() {
+                // Reset ajax attempts counter for the new page view
+                if (_self.core) {
+                    let ajaxPlugin = _self.core.getPlugin("AjaxDependencyPlugin");
```

Comment:
Add a qualifying that this is the IAjaxMonitorPlugin `_self.core.getPlugin<IAjaxMonitorPlugin>("AjaxDependencyPlugin")` so that ajaxPlugin is identified as the correct type and therefore it will contain the resetAjaxAttempts function.